src/bindings/python/valhalla/__init__.py

```diff
@@ -0,0 +1,4 @@
+from .actor import Actor
+from .config import get_config, get_help
```

Comment:
sadly it's hard to see the diff here. I had to change the directory for the python code, so this seems like new files. in the end I mostly added `get_help` here, which will print the help for any config